models/clipper/ethereum/clipper_v1_ethereum_trades.sql

```diff
@@ -0,0 +1,104 @@
+{{ config(
+    schema = 'clipper_v1-3_ethereum',
```

Comment:
any reason we don't follow the standard set in uniswap for a schema per version?
```clipper_v1_ethereum``` on the v1 model file here, same idea for 2 & 3 files

Reply:
schema is the same for v1 to v3, so I wanted to reuse.
Will duplicate it if that doesn't work.